### ISSUE TITLE
zone break spell crashes server (ATM 3)

### ISSUE THREAD

**Issue by RaineriZeppeli** [TeamWizardry/Wizardry#279]
I'm hosting an ATM3 server and one of our members started Wizardry, made a zone break spell and whenever it's cast, the server crashes. I've got 2 crash logs which are pretty similar.

[crash-2019-06-02_04.23.31-server.txt](https://github.com/TeamWizardry/Wizardry/files/3244357/crash-2019-06-02_04.23.31-server.txt)
[crash-2019-06-02_04.16.09-server.txt](https://github.com/TeamWizardry/Wizardry/files/3244358/crash-2019-06-02_04.16.09-server.txt)

Please fix i don't want them having a crash server button, thanks

**Comment by RaineriZeppeli** [comment]
We did some testing, 

Arc Illuminate works
Arc Break works
Zone Illuminate works

the only combination that seems to be crashing is zone break.

Also when any of the spells are created the command line reports a cache miss for that spell ring.

And when zone break is cast (on the ground) it reports a cache miss and then floods my command line with basically every mod erroring out and the server stopping. That's all we got, hope it helps to identify the problem.
